src/bo4e/bo/marktlokation.py

```diff
@@ -40,28 +45,31 @@ def _validate_marktlokations_id(self, marklokations_id_attribute, value):
                 f"The marktlokations_id '{value}' has checksum '{actual_checksum}' but '{expected_checksum}' was expected."
             )
 
+    # required attributes
+    bo_typ: BoTyp = attr.ib(default=BoTyp.MARKTLOKATION)
     marktlokations_id: str = attr.ib(validator=_validate_marktlokations_id)
     sparte: Sparte
     energierichtung: Energierichtung
     bilanzierungsmethode: Bilanzierungsmethode
+    netzebene: Netzebene
+
+    # optional attributes
     verbrauchsart: Verbrauchsart = attr.ib(default=None)
     unterbrechbar: bool = attr.ib(default=None)
```

Comment:
```suggestion
    unterbrechbar: Optional[bool] = attr.ib(default=None)
```
wenn es nullable ist.

Reply:
also weil das Attribut `unterbrechbar` durch den default-wert auch None werden kann, nimmt man das `Optional` dazu?

Reply:
ja, weil bool allein sollte nicht None sein.